Homework8/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const nunjucks = require("nunjucks");
+const mongoose = require("mongoose");
+const cookieSession = require("cookie-session");
+const path = require("path");
+const routers = require("./routers");
+const fs = require("fs");
+require("dotenv").config();
+
+const server = express();
+
+nunjucks.configure(path.join(__dirname, "templates"), {
+    watch: true,
+    autoescape: true,
+    express: server
+});
+
+mongoose.connect("mongodb://localhost:27017/chat8", {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useCreateIndex: true
+});
+
+mongoose.connection.on("error", err => {
+    console.error("Mongo error:", err);
+    process.exit(1);
+});
+
+mongoose.set("debug", true);
+
+server.use(cookieSession({
+    name: "session",
+    keys: [
+        "SecretKey1234567890"
```

Comment:
```suggestion
        process.env.SECRET || "SecretKey1234567890"
```
этот хардкод тоже не должен быть на конечном сервере